test/api_getProduct_test.dart

```diff
@@ -1858,4 +1859,60 @@ void main() {
     expect(result.product?.packagingTags, ['de:in-einer-plastikflasche']);
     expect(result.product?.quantity, '5.5 Liter');
   });
+
+  group('no nutrition data', () {
+    // This is barcode refers to a test product
+    const String barcode = '111111555555';
+
+    uploadProduct({required bool? noNutritionData}) =>
```

Comment:
* Explicit `Future<void>` would make it easier to read.
* `bool` instead of `bool?` would make it easier to read.